_component_design/vertical-stepper.md

```diff
@@ -4,31 +4,254 @@ parent: Content Areas
+![]({{site.url}}/assets/images/components/content-areas/vert-stepper/vert-stepper-spacing.png){: width="903px"}
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol-12 hxCol-xs-12 hxCol-sm-12 hxCol-md-12 hxCol-lg-12" %}
+
+### Specifications
+
+{% figure [caption:"Spacing specifications for the vertical stepper pattern"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/content-areas/vert-stepper/vert-stepper-specs.png){: width="903px"}
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Stepper counter
+
+The stepper counter displays the sequential number of a step as well as the status of the step. 
+
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+
+{% figure [caption:"Stepper counter states and specifications"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/content-areas/vert-stepper/vert-stepper-counter.png){: width="580px"}
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Clickable area
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+
+{% figure [caption:"Stepper expand / collapse clickable areas"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/content-areas/vert-stepper/vert-stepper-clickable.png){: width="903px"}
 {% endfigure %}
 
+{% endcolumn %}
+
+</div>
+
 </section>
 
 <section class="static-section" markdown="1">
 
-## Step 2
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol-12 hxCol-xs-12 hxCol-sm-12 hxCol-md-12 hxCol-lg-12" %}
+
+### Loading state
 
-{% figure [caption:""] [class:"image bg-light border"] %}
-![]({{site.cdn_url}}/img/components/stepper-2.svg){:width="100%"}
+{% figure [caption:"Stepper loading state"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/content-areas/vert-stepper/vert-stepper-loading.png){: width="903px"}
 {% endfigure %}
 
+{% endcolumn %}
+
+</div>
+
+</section>
+
+## Variances
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Submission controls
+
+As the stepper pattern isolates each sequential step within a multistep process a variance in the buttons displayed within each step will become apparent to the user. Dependent on which sequential step is displayed the following button(s) may be displayed.
```

Comment:
s/`displayed the following button(s)`/`displayed, the following buttons`

Reply:
updated